### PR TITLE
add tutorial blueprint

### PR DESCRIPTION
**Pull request by maimon33**
Please review and LMK if anything is missing

### CODE REVIEW

File: blueprint/inputs/inputs.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## This is an inputs file for the aws-ec2-blueprint.yaml. Many of these inputs have default values in the blueprint itself.
```
Comment (maimon33) [inline blueprint/inputs/inputs.yaml]: fix

File: blueprint/scripts/ansible/start.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+INVENTORY_FILE=$(ctx download-resource-and-render resources/inventory)
+cp $INVENTORY_FILE $INVENTORY_PATH
+rpx repl -p $INVENTORY_PATH -r HOST -w $tutorial_application_host_public_ip
+
+# Download the playbook that will download the roles for the other modules
+PLAYBOOK=$(ctx download-resource-and-render resources/${PLAYBOOK_FILENAME})
+cp $PLAYBOOK $PLAYBOOK_PATH
+
+# Manipulate playbook to prepare for Ansible run
+rpx repl -p $PLAYBOOK_PATH -r 'Ansible veriable sys_update' -w 'echo "{{ sys_update }}"'
```
Comment (maimon33) [inline blueprint/scripts/ansible/start.sh]: no capitals, no spaces, only underscores

File: blueprint/tutorial-blueprint.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      instance_type: { get_input: tutorial_application_server_instance_type }
+    relationships:
+      - target: keypair
+        type: cloudify.aws.relationships.instance_connected_to_keypair
+      - target: tutorial_application_server_ip
+        type: cloudify.aws.relationships.instance_connected_to_elastic_ip
+      - target: tutorial_application_security_group
+        type: cloudify.aws.relationships.instance_connected_to_security_group
+
+  keypair:
```
Comment (maimon33) [inline blueprint/tutorial-blueprint.yaml:102]: name

File: blueprint/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# cloudify-interactive-tutorial
+
+## Summary
+
+Deploys Cloudify Interactive tutorial Application using cfy local on AWS.
+
+A bit more detailed explanation:
```
Comment (nir0s) [inline blueprint/README.md:7]: wow... too detailed :)

Please remove up until "This blueprint consists of"...

File: blueprint/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+The blueprint in this repository sets up interactive tutorial application on AWS infrastructure. AWS plugin will initiate the instance on Europe West region and the application code will be deployed by Ansible
+
+
+This blueprint consist of: Keypair, ElasticIP, Security Group and Compute instance. The tutorial application will eventually be hosted on the Compute instance.
+Prior to running the blueprint you'll need to make some adjustments to the input file regarding the AWS environment.
+
+### Set up an AWS environment:
+
+You'll need to make some adjustments to the input file to provide your own AWS credentials.
+Edit inputs/inputs.yaml and add you access key and secret
```
Comment (nir0s) [inline blueprint/README.md]: you -> your

File: blueprint/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+You can also change the region, instance type and AMI (AMI must be support hvm)
```
Comment (nir0s) [inline blueprint/README.md]: must support. no "be"

File: blueprint/README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+
+To see the application, you can go to http://[THE APPLICATION URL]:8088/.
+
+To uninstall the application run:
+```bash
+cfy local execute -w uninstall --task-retries=9 --task-retry-interval=10
+```
+
+### Once server is deployed you should consider closing port 22 that was used by Ansible.
```
Comment (nir0s) [inline blueprint/README.md]: Once THE server is deployed you MUST close port 22 manually in the security group

**Pull request closed without merging**